docker/setup.bat

```diff
@@ -0,0 +1,94 @@
+@echo off
+setlocal
+
+echo Welcome to Endatix Platform! This script will help you quickly set up the platform using Docker. For additional details and troubleshooting, please check README.md.
+echo.
+
+REM Check if Pester is loaded in PowerShell and remove it if present
+powershell -Command "if (Get-Module -Name Pester -ErrorAction SilentlyContinue) { Remove-Module -Name Pester -Force; Write-Host 'Pester has been disabled because this script cannot run with it enabled.' }"
+
+REM Check if Docker is installed
+docker --version >nul 2>&1
+IF %ERRORLEVEL% NEQ 0 (
+    echo Docker is not installed. Please install Docker before proceeding.
+    pause
+    exit /b 1
+)
+
+REM Check if docker-compose.dev.yml exists in the current directory
+IF NOT EXIST "docker-compose.dev.yml" (
+    echo docker-compose.dev.yml not found in the current directory. Please ensure it is present.
+    pause
+    exit /b 1
+)
+
+echo Docker and Docker Compose are installed, and docker-compose.dev.yml is found.
+
+REM Create or overwrite the .env file
+echo Creating .env file and setting the necessary environment variables...
+echo # Environment variables for Endatix > .env
+
+REM Enable delayed expansion for data collection
+setlocal enabledelayedexpansion
+
+REM Prompt for environment variables with default values if nothing is entered
+set /p ASPNETCORE_ENVIRONMENT="Enter ASPNETCORE_ENVIRONMENT (skip to use the default value 'Development'): "
+IF "%ASPNETCORE_ENVIRONMENT%"=="" set "ASPNETCORE_ENVIRONMENT=Development"
+
+set /p SECURITY_JWT_SIGNING_KEY="Enter SECURITY_JWT_SIGNING_KEY (JWT signing key to be used in Endatix API, skip to use the default value): "
+IF "%SECURITY_JWT_SIGNING_KEY%"=="" set "SECURITY_JWT_SIGNING_KEY=L2yGC_Vpd3k#L[<9Zb,h?.HT:n'T/5CTDmBpDskU?NAaT$sLfRU"
```

Comment:
@dspetrov  - it's much better, intuitive and faster now - definitely makes a difference!

Only one issue on my Mac I have this error:

`failed to read ../../Code/endatix/endatix/docker-test/.env: line 3: unexpected character "/" in variable name "T/5CTDmBpDskU?NAaT$sLfRU'"`

Can you remove the / from the default key. It work for me with it removed?